include/NovelRT/Maths/GeoBounds.h

```diff
@@ -10,25 +10,85 @@
 
 namespace NovelRT::Maths
 {
+    /**
+     * @brief Represents a bounding box in 2D space with rotational support.
+     */
     class GeoBounds
     {
     public:
+        /// @brief The centre of the GeoBounds instance as a position in X,Y coordinates in the space used during its creation.
         GeoVector2F position;
+        /// @brief The size represented by Width and Height. This is subject to the space used during the creation of the GeoBounds instance.
         GeoVector2F size;
+        /// @brief The rotation of the GeoBounds instance represented in Degrees.
```

Comment:
Does this type still use degrees instead of radians?

If so we have a bug to fix lol

Reply:
Looking at the implementation it calls `Vector2F::rotateToAngleAroundPoint(float, GeoVector2F)` with the rotation field directly.

looking at the implementation for that function...
```cpp
void rotateToAngleAroundPoint(float angleRotationValue, GeoVector2F point) noexcept
{
    *reinterpret_cast<glm::vec2*>(this) =
        glm::rotate((*reinterpret_cast<glm::vec2*>(this) = *reinterpret_cast<const glm::vec2*>(this) -
                                                           *reinterpret_cast<const glm::vec2*>(&point)),
                    glm::radians(angleRotationValue)) +
        *reinterpret_cast<const glm::vec2*>(&point);
}
```
it seems to call `glm::radians(float)` which [”Converts degrees to radians and returns the result."](https://glm.g-truc.net/0.9.4/api/a00136.html#ga4fb76e28851c9ff6653532566084e091), so i assume it is still using degrees.

with that said, what would be the best thing for me to do with commenting this?